product/reports/780_Tenants - Tenant Quotas/001_Tenant Quotas.yaml

```diff
@@ -6,6 +6,11 @@ rpt_group: Custom
 priority: 231
 rpt_type: Custom
 db: Tenant
+conditions: !ruby/object:MiqExpression
```

Comment:
Glad this works! @gtanzillo could there be a more efficient way to do this within MiqExpressions?